app/views/patron_requests/login.html.erb

```diff
@@ -89,7 +89,8 @@
                 <details>
                     <summary class="btn btn-link p-0<% unless @guest_can_request %> disabled<%end %>">Proceed as visitor <i class="bi bi-chevron-right"></i></summary>
                     <% if @guest_can_request %>
-                        <%= form_tag new_patron_request_path(new_params), method: 'POST', class: 'form p-3', data: { turbo: false } do %>
+                        <%= form_tag register_visitor_url, method: 'POST', class: 'form p-3', data: { turbo: false } do %>
+                            <%= hidden_field_tag :referrer, new_patron_request_path(new_params.merge(step: 'select')) %>
                             <div class="form-group mt-2">
```

Comment:
Originally, the new_patron_request_path had "step=select" as a parameter.  Just want to check that we still want this parameter, since the form tag in the current code doesn't include "merge(step:'select')".  This has been moved to the referrer field because we want to submit the form to the session controller